### PR TITLE
Failover recursive, without scripts

### PR DESCRIPTION
**Pull request by jsalonl**

### CODE REVIEW

**Comment by eworm-de** [comment]
Can you explain what this is supposed to do and why you think this should be added to the repository?

File: route-failover [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+:global Gateway2 "2"
+#DNS OpenDNS // Change your own hosts
+:global Host1 "208.67.222.222" 
+:global Host2 "208.67.220.220"
+/ip route
+add dst-address=0.0.0.0/0 gateway=$Gateway1 distance=1 check-gateway=ping
+add dst-address=0.0.0.0/0 gateway=$Gateway2 distance=2
+
+/ip route
+add dst-address=$Host1 gateway=$Gateway1 scope=10
```
Comment (mtrimarchi) [inline route-failover:13]:
```suggestion
add dst-address=$Host1 gateway=$Gateway1 scope=30 target-scope=30
```

File: route-failover [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+add dst-address=$Host2 gateway=$Gateway2 scope=10
```
Comment (mtrimarchi) [inline route-failover:14]:
```suggestion
add dst-address=$Host2 gateway=$Gateway2 scope=30 target-scope=30
```

**Review by mtrimarchi — CHANGES_REQUESTED** [review]
@jsalonl you didn't specified a value of 30 for the route `scope` and `target-scope`!

**Comment by mtrimarchi** [comment]
@eworm-de this is a failover configuration using [recursive next hop resolving ](https://rickfreyconsulting.com/recursive-next-hop-resolving/) technique.

**Comment by eworm-de** [comment]
My repository is supposed to hold scripts that are installed to the device for regular usage.

You posted a configuration snippet. It is applied to configuration, not run regularly. Thus it is not applicable.

Probably something like this is better kept in a personal repository for configuration snippets.

**Pull request closed without merging**